app/gui2/shared/languageServerTypes.ts

```diff
@@ -386,9 +386,7 @@ export namespace response {
     contentRoots: ContentRoot[]
   }
 
-  export interface FileContents {
```

Comment:
actually i discovered this was *in*correct, when i was implementing this:

https://github.com/enso-org/enso/blob/2d2c9c9261030de504edf7a976fd36daea6b9715/app/gui2/ydoc-server/languageServerSession.ts#L585-L596

on that note... i'm not 100% sure about this. it doesn't affect the fix if it's removed - it's entirely possible that I've misunderstood what `reload` is supposed to do

Reply:
note that there *is* a `FileContents` type: https://github.com/enso-org/enso/blob/develop/docs/language-server/protocol-language-server.md#filecontents

also note that `*Result` types were basically only added so that the `Result` interfaces are valid TS code - in GUI2 these should correspond to `response.*`